integration_tests/e2e/newCourtCaseJourney.cy.ts

```diff
@@ -1,26 +1,6 @@
 import Page from '../pages/page'
 import StartPage from '../pages/startPage'
-import CourtCaseReferencePage from '../pages/courtCaseReferencePage'
```

Comment:
the nav has entirely changed in the journey with the introduction of a central task list page where the user keeps going back to that page, Will keep uncommenting this file when the new tickets are implemented